SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -480,6 +480,356 @@ public void UpdatePerson(UpdatePersonRequest request)
             _databaseContext.SaveChanges();
         }
 
+        public void PatchPerson(PatchPersonRequest request)
+        {
+            Person person = _databaseContext
+                .Persons
+                .Include(x => x.Addresses)
+                .Include(x => x.PhoneNumbers)
+                .Include(x => x.OtherNames)
+                .Include(x => x.KeyContacts)
+                .Include(x => x.GpDetails)
+                .Include(x => x.TechUse)
+                .Include(x => x.Disability)
+                .Include(x => x.Emails)
+                .Include(x => x.LastUpdated)
+                .FirstOrDefault(x => x.Id == request.Id);
+
+            if (person == null)
+            {
+                throw new UpdatePersonException("Person not found");
+            }
+
+            if (request.ContextFlag != null)
```

Comment:
Have to be careful here that the client doesn't pass empty strings instead of nulls to avoid unintentional data loss.